### DISCUSSION TITLE
neue AVM Source-Updates: 7530ax + 6890, Finals

### DISCUSSION THREAD

**Discussion by maverick3005** [Freetz-NG/freetz-ng#782] [Allgemein]
auf osp.avm.de liegen jetzt auch die neuen Sourcen für 7530ax + 6890
- Wie gehabt nichts für die Inhaus/Labore:

https://osp.avm.de/fritzbox/fritzbox-6890-lte/
https://osp.avm.de/fritzbox/fritzbox-7530-ax/

Wer also gerne selber etwas basteln will... viel Spass...

**Comment by fda77** [comment]
Das sind ja nur 07.29 und .31 Sourcen . Fehlen die in freetz?
